src/utils/streams.js

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env node
+
+var readline = require('readline');
+var fs = require('fs');
+var csv = require('csvtojson');
+var program = require('commander');
+
+var reader = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    terminal: false
+});
+
+function validateOnDefinitions(filePath) {
```

Comment:
Please extract validators to separate utils file and then import them as `import { validateOnDefinitions, validateFileExtension, validateOnFileExist } from './validation'`